libs/ui/src/lib/notification/notification.component.html

```diff
@@ -19,14 +30,17 @@ <h2 class="cvi-notification__title">{{ title }}</h2>
     </div>
     <div class="cvi-notification__content" dataAttribute="content">
       <ng-container *ngIf="size === 'compact' && getIconName() as iconName">
-        <cvi-ng-labeled-icon [name]="iconName" [iconHeight]="14" class="cvi-notification__content-icon-wrapper" iconClass="cvi-notification__content-icon" dataAttribute="icon">
+        <cvi-ng-labeled-icon [name]="iconName" [iconHeight]="24" class="cvi-notification__content-icon-wrapper"
+                             iconClass="cvi-notification__content-icon" dataAttribute="icon">
           <ng-container *ngTemplateOutlet="content"></ng-container>
         </cvi-ng-labeled-icon>
       </ng-container>
+
       <ng-container *ngIf="size !== 'compact' || !getIconName()">
         <ng-container *ngTemplateOutlet="content"></ng-container>
       </ng-container>
       <ng-template #content>
+        <b *ngIf=contentPrefix> {{contentPrefix}}: </b>
```

Comment:
@Seeru-crypto Should it be a separate prop? Or maybe it makes sense to insert `title` prop value here?

Reply:
would you elaborate a bit more?
The message prefix is a bit different from the standard title, which is used here. 
Therefore I created a separate prop for it.

Reply:
> The message prefix is a bit different from the standard title, which is used here.
If that's the case, then feel free to go with the separate prop.